cdc/sorter/leveldb/message/task.go

```diff
@@ -44,6 +45,9 @@ type Task struct {
 	// Deletes all of the key-values in the range.
 	// reader -> leveldb and leveldb -> compactor
 	DeleteReq *DeleteRequest
+
+	// A test message.
+	Test *Test
```

Comment:
how about leave a todo message there that `remove test logic later`?

Reply:
I don't think they will be removed, I found them are quite handy for testing😅